README.md

```diff
@@ -111,55 +111,20 @@ Ingress resources need to specify the following in their annotations for them to
 
     kubernetes.io/ingress.class: alb
 
-This is automatically configured for the charts in this repository.    
+This is automatically configured for the charts in this repository.
 
 **Important**: When the cluster is (re)deployed, there may be a delay in the DNS changes propagating to all hosts on
 the internet. If redeploying the infrastructure to another region, **make sure the old deployment still exists**.
 There can only be one deployment bound to the domain at any given time. Destroying the old deployment before creating
 the new one will result in downtime.
 
-Deployments
------------
-
-Deployments refer to the instances of various  services running to use the single-cell pipeline, e.g. `ui`, `api`, `worker`
-copies. Deployments are managed *automatically* by [Flux](http://fluxcd.io/).
-
-### Flux
-
-Flux is a GitOps Continuous Delivery platform. It is deployed on the cluster alongside all other base infrastructure. It
-manages custom resources called *HelmRelease*. A HelmRelease contains data identifying the Helm Chart that is to be deployed,
-as well as any custom configuration to the chart that may be necessary. Broadly, it is designed to automate manual
-`helm install` or `helm upgrade` operations.
-
-Flux is granted read and write access to this repository by the deploy script. This is by automatically creating/updating a
-[deploy key](https://github.com/hms-dbmi-cellenics/iac/settings/keys) for this repository by the base infrastructure deployment
-workflow.
-
-### Releases
-
-Flux running on the `$ENVIRONMENT` cluster is continuously scanning the `releases/$ENVIRONMENT/` folder in this directory
-for updates. When a change to a file in the `releases/$ENVIRONMENT/` folder is found, Flux will automaticlly apply those
```

Comment:
can you add this and any other information you think is relevant as a README to the releases repo?

Reply:
And in this repository in particular,I would keep the section "Deployments" and add that these are managed by the `releases` repository